src/components/request/RequestView.tsx

```
@@ -7,7 +11,29 @@ const RequestView = ({
 }: RoutineRequestDetail) => {
-  const closeModal = useModalStore((state) => state.close);
+  const user = useAuthStore((state) => state.user);
+  const [comment, setComment] = useState('');
+  const replyRequest = useReplyRequestMutation(user);
+
+  const handleSubmit = async (status: CheckStatus) => {
+    try {
+      await replyRequest.mutateAsync({
+        confirmId: id,
+        checkStatus: status,
+        checkComment: commnet,
```

ESLint

[error] 'commnet' is not defined.

(no-undef)